mixins/localize-mixin.md

```diff
@@ -56,27 +58,57 @@ class MyComponent extends LocalizeStaticMixin(LitElement) {
 
 }
 ```
-
 #### Example 2: Dynamically Imported Resources
 
 This approach is better for components with many language resources. By importing them dynamically, only the resources for the requested language are actually fetched and downloaded.
 
 Store your resources in individual files, one for each language:
 ```javascript
 // en.js
-export const val = {
+export default {
```

Comment:
I think this is going to be a breaking change for those using the Serge integration. Was this change just ergonomic? Or I guess is this what the `exportName` config value is for?

Reply:
I don't think serge cares. I _believe_ it is just skipping over it with its regex anyway looking for the key/value pairs. I'll confirm with Sean. Just ergonomic and searching this repo I found 143 `export default` and only 30 `export const` in `lang` directories so I figured it was a good default.

Reply:
Cool yeah agreed, and even better if it doesn't impact Serge.

Reply:
Confirmed it doesn't care. But they need to be consistent between languages, obviously, which this will enforce as well.

Reply:
Awesome, thanks for checking that.